src/plugins/gr-he-common/vm/cloud_init.py

```diff
@@ -929,11 +933,34 @@ def _customization(self):
                     default=_('No')
                 ) == _('Yes').lower()
 
+            if (
+                self.environment[
+                    ohostedcons.CloudInit.APPLY_OPENSCAP_PROFILE
+                ] and self.environment[
+                    ohostedcons.CloudInit.OPENSCAP_PROFILE_NAME
+                ] is None
+            ):
+                self.environment[
+                    ohostedcons.CloudInit.OPENSCAP_PROFILE_NAME
+                ] = self.dialog.queryString(
+                    name='CI_OPENSCAP_PROFILE_NAME',
+                    note=_(
+                        'Please provide the security profile you '
+                        'would like to use (@VALUES@) [@DEFAULT@]: '
+                    ),
+                    prompt=True,
+                    validValues=(_('stig'), _('pci-dss')),
```

Comment:
I don't think we want to localize here. IIUC translated input will be saved and passed as-is, and break both your test below and ansible code. If we do want to localize, we have to save the English strings to the env somehow, e.g.:

```
   nativeValidValues = ('stig', 'pci-dss')
   vvDict = { _(val):val for val in nativeValidValues
...
...
   env[PROFILE_NAME] = vvDict(
     queryString(
...
       validValues = vvDict.keys()
    )
   )
```

But I don't think this is needed.